approaches/bulk-data-loading/src/GqlMutations.js

```diff
@@ -0,0 +1,85 @@
+export class GqlMutations {
+
+    static createInventoryListQuery() {
+        let query = `
+        query ListInventoryItems {
+          list_InventoryItems(readMode: NODE_LEDGERED) {
+            _InventoryItems {
+              _id
+            }
+            nextToken
+          }
+        }
+    `;
+
+        return {
+            query: query,
+            variables: {}
+        }
+    }
+
+    static createInventoryListNextPageQuery(nextToken) {
```

Comment:
Same as above, we should be able to use the built-in client function for this

Reply:
Done